server/halomod_app/routes/plot_types.py

```diff
@@ -0,0 +1,21 @@
+from halomod_app import utils
+from flask import Blueprint, jsonify
+
+plot_types_bp = Blueprint('plot_types', __name__)
+
+
+# This endpoint returns the details of all the different plot types that
+# can be used to represent a halo model.
+#
+# expects: None
+# outputs: {
+#   xLabels: <data from utils.py>,
+#   plotOptions: <data from KEYMAP in utils.py>
+# }
```

Comment:
Maybe make this a docstring for consistency?

Reply:
Oh, must have missed that. Will do.

Reply:
That one has been fixed 😄